.travis/test.sh

```diff
@@ -78,13 +71,6 @@ then
   UT_FLAGS+=" -K crypto -K not_pypy"
 fi
 
-if python --version 2>&1 | grep -q '^Python 3\.'
```

Comment:
Why did you remove these lines?

Reply:
They are only used with this test.

Reply:
I think that we can only comment the failing test and keep the tags are they are today. It will help us track the fact that we need to fix the test =)

Reply:
Could you keep the changes to the regressions.uts ?

Reply:
Done